pkg/apis/autoscaling/v1alpha1/pa_lifecycle_test.go

```diff
@@ -424,7 +424,7 @@ func TestIsActivating(t *testing.T) {
 	}{{
 		name:         "empty status",
 		status:       PodAutoscalerStatus{},
-		isActivating: false,
+		isActivating: true,
```

Comment:
why this?

Reply:
Well in general the logic is assuming that if we're not ready, we're status Unknown. In this case Activating is the only status, and empty is a pretty weird case - we want to ensure that we are considering this case to be "unknown" in terms of activating (because we haven't observed anything if defaults aren't set).

However we _*do*_ always initialize conditions, so this is a case we never expect to see at all. The choice is a bit arbitrary.

Reply:
The alternative:

```
	cond := pas.GetCondition(PodAutoscalerConditionActive)
	return cond != nil &&  cond.IsUnknown()
```

is sort of awkward. I'd prefer to consider the 'nil' case as unknnown.

Reply:
This needs reverting?